internal/integration/integration_test.go

```diff
@@ -225,6 +225,12 @@ func TestIntegration(t *testing.T) {
 		}
 	}
 
+	// Rotate Keys
+	time.Sleep(2 * time.Second) // Ensure DeleteOldKeyPeriod is elapsed
```

Comment:
should probably do this before we publish keys and then verify that we god a revision token back from the publish.

Actually using that revision token would be part of the upcoming revision test.

Reply:
sure we can do that. Really we want to rotate them and then assert that an old one is rejected

Reply:
but at the least this adds coverage from a client-request perspective. We can also see from the logger that the thing happens.

Reply:
yeah, that would be great

I think that the new one isn't "rejected" but it acts as if you didn't pass one at all